analytics_data_api/v0/models.py

```diff
@@ -249,9 +249,19 @@ def get_last_updated(cls):
 class RosterEntry(DocType):
 
     course_id = String()
+    user_id = Integer()
```

Comment:
I think I asked this question in a google doc somewhere, but have forgotten the answer.  What is `user_id`?  How it is different from `username`?